ui/components/app/gas-customization/advanced-gas-inputs/advanced-gas-inputs.component.js

```diff
@@ -21,6 +21,7 @@ export default class AdvancedGasInputs extends Component {
     minimumGasLimit: PropTypes.number,
     customPriceIsExcessive: PropTypes.bool,
     networkSupportsSettingGasPrice: PropTypes.bool,
+    disabled: PropTypes.bool,
```

Comment:
Why were the changes in this component made? It looks like this prop is never set.

Reply:
These crept in after the last rebase....I am going to compare with previous commits and fix.